node_modules/game-iverse/brickbreak/brickbreakclasses.js

```diff
@@ -0,0 +1,189 @@
+                        this.ctx.fill();
+                        this.ctx.closePath();
+                    }
+                }
+            }
+        }
+
+        this.collisionDetection =function(ball, hud) {
+            for (var c = 0; c < this.brickColumnCount; c++) {
+                for (var r = 0; r < this.brickRowCount; r++) {
+                    var b = this.bricks[c][r];
+                    if (b.status === 1) {
+                        if (ball.x > b.x && ball.x < b.x + this.brickWidth && ball.y > b.y && ball.y < b.y + this.brickHeight) {
+                            ball.dy = -ball.dy;
+                            b.status = 0
+                            hud.score++
+                            if (hud.score == this.brickColumnCount * this.brickRowCount) {
+                                alert("YOU WIN, CONGRATULATIONS!");
+                                document.location.reload();
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+}
+
+export class Paddle {
+    constructor (world) {
+        this.canvas = world.canvas;
+        this.ctx = world.ctx;
+        this.paddleHeight = 10;
+        this.paddleWidth = 75;
+        this.paddleX = (this.canvas.width - this.paddleWidth) / 2; // location
+        this.rightPressed = false;
+        this.leftPressed = false;
+        this.keyDown = document.addEventListener("keydown", (e) => this.keyDownHandler(e), false);
+
+        this.keyUp = document.addEventListener("keyup", (e) => this.keyUpHandler(e), false);
+
+        this.mouseMove = document.addEventListener("mousemove", (e) => this.mouseMoveHandler(e), false);
+    }
+    keyUpHandler(e) {
+        if (e.keyCode == 39) {
+            this.rightPressed = false;
+        }
+        else if (e.keyCode == 37) {
+            this.leftPressed = false;
+
+        }
+    };  
+
+    keyDownHandler(e) {
+        if (e.keyCode == 39) {
+            this.rightPressed = true;
+        }
+        else if (e.keyCode == 37) {
+            this.leftPressed = true;
+
+        }
+    };
+
+    draw() {
+        this.ctx.beginPath();
+        this.ctx.rect(this.paddleX, this.canvas.height - this.paddleHeight, this.paddleWidth, this.paddleHeight);
+        this.ctx.fillStyle = "#0095DD";
+        this.ctx.fill();
+        this.ctx.closePath();
+    }
+
+    movement(){
+        if (this.rightPressed && this.paddleX < this.canvas.width - this.paddleWidth) {
+            this.paddleX += 7;
+        } else if (this.leftPressed && this.paddleX > 0) {
+            this.paddleX -= 7;
+        }
+    }
+
+    mouseMoveHandler(e){
+        var relativeX = e.clientX - this.canvas.offsetLeft;
+        if (relativeX > 0 && relativeX < this.canvas.width) {
+            this.paddleX = relativeX - this.paddleWidth / 2
+        }
+    };
+}
+
+// module.exports = {
+//     World: World,
+//     Hud:Hud,
+//     Brick:Brick,
+//     Ball:Ball,
+//     Paddle:Paddle
+// }
```

Comment:
@Roodiger why does your version have a `node modules/` folder with our package inside of it? This whole repo is the npm package and should not have our repo inside of it. Please remove it. 

The idea is to supply building blocks for others to use. I should have clarified that the html and js file you were supplying was for testing purposes only, and would be removed and added to a read me file upon merging. If you want to test your package create a test repo to `npm link` your two repos together.